content/support.rst

```diff
@@ -0,0 +1,193 @@
+Support
+#######
+
+:menu_order: 009
+
+.. contents::
+
+
+Introduction
+============
+
+This page describes the guidelines for the support activities together with
+best practices that were extracted based on our past experience with support
+related activities.
+
+For access to any of the Support platforms, please contact the Support team.
+
+For the full details on the Support offering, please go to the website's
```

Comment:
I don't think that this link should be part of the styleguide.

If we want to describe bests ways to offer support, that should be done inside the styleguide